wolfcrypt/src/evp.c

```diff
@@ -5431,6 +5431,78 @@ int wolfSSL_EVP_MD_type(const WOLFSSL_EVP_MD *md)
         return ret;
     }
 
+    /* WOLFSSL_SUCCESS on ok */
+    int wolfSSL_EVP_DigestFree(WOLFSSL_EVP_MD_CTX* ctx)
```

Comment:
Why would we add a non standard OpenSSL Compatibility API for freeing?  An existing app using OpenSSL would not be aware of this.  Also, we already have a standard API

wolfSSL_EVP_MD_CTX_cleanup()

Why doesn't that work?

Reply:
@toddouska : Thanks for spotting that! I've changed this to use the existing `EVP_MD_CTX_cleanup` and `EVP_CIPHER_CTX_cleanup`.